leadership.html

```diff
@@ -149,10 +158,20 @@ <h1 class="heading">Ian Roth </h1>
             class="image-2" />
         <div class="w-layout-blockcontainer container-5 w-container">
             <h1 class="heading">Scott Whalen</h1>
-            <p class="paragraph"><strong>Power Train Lead/Secretary<br /></strong>Sophomore Electrical Engineer<br />
+            <p class="paragraph"><strong>Power Train Lead<br /></strong>Sophomore Electrical Engineering<br />
             </p>
         </div>
     </div>
+    <div class="w-layout-blockcontainer container-4 w-container"><img
+        src="./images/65CD4C1C-6295-4ADA-81E1-04E34C8BA8B8.jpg"
+        loading="lazy" width="120" sizes="120px" alt=""
+        class="image-2" />
+    <div class="w-layout-blockcontainer container-5 w-container">
+        <h1 class="heading">Beyza Inal</h1>
+        <p class="paragraph"><strong>Secretary<br /></strong>Sophomore Computer Engineering<br />
+        </p>
+    </div>
+</div>
```

Comment:
Position change probably will come but since not powertrain lead, put after treasurer